.github/workflows/ci.yml

```diff
@@ -86,4 +86,4 @@ jobs:
         pip freeze
     - name: Run test
       run: |
-        CI=1 FUNSOR_BACKEND=jax make test
+        CI=1 JAX_ENABLE_X64=1 FUNSOR_BACKEND=jax make test
```

Comment:
@fehiepsi does this seem ok to you? I'm working around errors of the form
```
UserWarning: Explicitly requested dtype float64 requested in full is not available, and will be truncated to dtype float32. To enable more dtypes, set the jax_enable_x64 configuration option or the JAX_ENABLE_X64 shell environment variable. See https://github.com/google/jax#current-gotchas for more.
```

Reply:
I can address it in a follow-up PR.